<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistogramSettings</class>
 <widget class="QDialog" name="HistogramSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>219</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Histogram Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Y Axis</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="checkBoxYAxisLogScale">
        <property name="text">
         <string>Log scale</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Maximum value</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QRadioButton" name="radioButtonDataMaxValue">
           <property name="text">
            <string>Data maximum value</string>
           </property>
           <attribute name="buttonGroup">
            <string>buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QRadioButton" name="radioButtonYValue">
              <property name="text">
               <string>Value</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string>buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="LineEditFocus" name="lineEditValue"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEditFocus</class>
   <extends>QLineEdit</extends>
   <header>LineEditFocus.h</header>
   <slots>
    <signal>focusIn()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>radioButtonYValue</tabstop>
  <tabstop>lineEditValue</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HistogramSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HistogramSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonDataMaxValue</sender>
   <signal>toggled(bool)</signal>
   <receiver>HistogramSettings</receiver>
   <slot>ToggledYMaxValue(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonYValue</sender>
   <signal>toggled(bool)</signal>
   <receiver>HistogramSettings</receiver>
   <slot>ToggledYValue(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditValue</sender>
   <signal>editingFinished()</signal>
   <receiver>HistogramSettings</receiver>
   <slot>EditingFinishedYValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditValue</sender>
   <signal>focusIn()</signal>
   <receiver>HistogramSettings</receiver>
   <slot>FocusInYValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditValue</sender>
   <signal>focusIn()</signal>
   <receiver>lineEditValue</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxYAxisLogScale</sender>
   <signal>toggled(bool)</signal>
   <receiver>HistogramSettings</receiver>
   <slot>ToggledYAxisLogScale(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ToggledYMaxValue(bool)</slot>
  <slot>ToggledYValue(bool)</slot>
  <slot>EditingFinishedYValue()</slot>
  <slot>FocusInYValue()</slot>
  <slot>ToggledYAxisLogScale(bool)</slot>
  <slot>ToggledXAxisLogScale(bool)</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
